src/machine/behavior/is/memory/stwu.rs

```diff
@@ -1,6 +1,7 @@
 use machine::state::State;
+use machine::behavior::is::memory::stw;
 
-pub fn stwu(_state: &mut State, _x: u8, _y: u8, _z: u8) {
-    unimplemented!();
+/// store wyde unsigned
+pub fn stwu(state: &mut State, x: u8, y: u8, z: u8) {
+    stw(state, x, y, z);
```

Comment:
In stbu, stwu, sttu and stou no overflow occurs. In stb, stw, stt and sto, an overflow can occur but the program does not do something absolutely strangein this case, so imho it is (for our project) not absolutely necessary to catch the overflow.